src/libs/blueprint/blueprint_mcarray.cpp

```diff
@@ -92,30 +115,75 @@ verify(const Node &n)
 bool verify(const conduit::Node &n,
             Node &info)
 {
+    bool res = true;
+
+    // mcarray needs to be an object or a list
+    if( ! (n.dtype().is_object() || n.dtype().is_list()) )
```

Comment:
This is an incredibly minor complaint, but I feel like this line doesn't match the coding standards used for of the rest of Conduit.  In particular, it doesn't feel like the expanded spacing in this statement fits; something like `if(!(n.dtype().is_object() || n.dtype().is_list()))` seems more appropriate.